rocAL/include/augmentations/geometry_augmentations/node_crop_resize.h

```diff
@@ -29,6 +29,10 @@ class CropResizeNode : public CropNode {
    public:
     CropResizeNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     CropResizeNode() = delete;
+    ~CropResizeNode() {
```

Comment:
Is this needed, those are shared pointers and they should be released, Please do not do manual reset as discussed